src/Microsoft.Health.Fhir.Liquid.Converter/Processors/BaseProcessor.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Health.Fhir.Liquid.Converter.Processors
     public abstract class BaseProcessor : IFhirConverter
     {
         private readonly ProcessorSettings _settings;
+        private const int _maxIterations = 100000;
```

Comment:
can help doc this limitation?

Reply:
Hi @ginalee-dotcom, could you please help create a PR to update the document about this change? The maximum iteration of for-loop in liquid templates is now set to 100,000. Previously there's no limitation about this. If the limitation is reached, an exception will be thrown with detailed message. This is to help make sure our engine returns a result in time.